processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractStaticParseToStringConversion.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.internal.conversion;
+
+import org.mapstruct.ap.internal.model.common.ConversionContext;
+
+/**
+ * Base class for a conversion between a type <tt>T</tt> and {@link String},
+ * where {@code T#parse(String)} and {@code T#toString} are inverse operations.
+ */
+public abstract class AbstractStaticParseToStringConversion extends SimpleConversion {
```

Comment:
I would get rid of this class and just do the needed conversion in it's implementations. Or would only have a single class where you pass in the class that needs to be used in the `getFromExpression`

Reply:
Imo, this class should stay -- or, at least the code sharing it offers. It's easy to imagine additional classes with the `parse`/`toString` property that mapstruct would want to support. Joda Duration is the easy example.
I'm open to moving it to a utility class instead of a base class. I was moreso following the [AbstractJavaTimeToStringConversion](https://github.com/mapstruct/mapstruct/blob/cf668bea77d5d48f34d32c229823a04dbd01f21d/processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java) example when making the base class.

Reply:
No problem for me